de.fu_berlin.inf.dpp.whiteboard/frontend/src/components/menu.jsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import Menus from './menus';
+import LOGIC from '../constants/logic';
+
+export default class Menu extends React.Component {
+  /**
+   * @param {Object} props
+   * properties given to the constructor
+   */
+  constructor(props) {
+    super(props);
+    this.toolManager = props.toolManager;
+    this.canvasResizer = props.canvasResizer;
+    this.state = {};
+    this.miniMode = false;
+    this.wrapper = document.getElementById("menuContainer");
+    //start in miniMode because the windows in eclipse are small
+    //this.switchMiniMode();
```

Comment:
Why is this uncommented?

Reply:
It would not be clear to the user that the menu actually has another normal mode, it would seem as if the menu is only restricted to this small size